### ISSUE TITLE
Resurrecting recipe on melpa ...

### ISSUE THREAD

**Issue by rprimus** [emacsattic/quack#1]
Mon May 28 00:42:04 BST 2018

Hi melpa team (@milkypostman, @purcell, @tarsius).

Am working with `quack.el`'s author Neil Van Dyke (@neilvandyke) to allow it to again be available via `melpa.org`.    (It was removed via https://github.com/melpa/melpa/commit/0047a17d4049270a568e87274e378eb70366c135.)

It should hopefully be soon hosted at github.

Is there anything else that should be done to allow it to be once again be available on melpa?

Will keep you updated via this issue.

**Comment by purcell** [comment]
Yep, just file a PR or let us know here when ready, and we'll take care of it.

**Comment by rprimus** [comment]
Tue Nov  6 11:21:10 GMT 2018

Hi again,

`quack.el` is now available (via secure connection) on the author's website:
https://www.neilvandyke.org/quack/quack.el

@tarsius:
Before making the pull request to (re)add the [original recipe](https://github.com/melpa/melpa/blob/094f63e451622571aac832b14221a0d5a96de9c5/recipes/quack) to melpa,  what needs to be done to
have `emacsmirror` host the file?   (Is this something I could do using `epkg`?)

(`quack.el` was moved to the attic via the following  commit:)
https://github.com/emacsmirror/epkgs/commit/eff2d0a7e8296784ebef0a37655daf7351dc69a9
